watertap/property_models/seawater_prop_pack.py

```diff
@@ -63,17 +63,38 @@
 )
 import idaes.core.util.scaling as iscale
 from watertap.core.util.scaling import transform_property_constraints
+from pyomo.common.config import ConfigValue, In
+from enum import Enum, auto
 
 # Set up logger
 _log = idaeslog.getLogger(__name__)
 
 
+class DiffusCalculationType(Enum):
+    isothermal = auto()
+    nonisothermal = auto()
+
+
 @declare_process_block_class("SeawaterParameterBlock")
 class SeawaterParameterData(PhysicalParameterBlock):
     """Parameter block for a seawater property package."""
 
     CONFIG = PhysicalParameterBlock.CONFIG()
 
+    CONFIG.declare(
+        "diffus_calc_type",
+        ConfigValue(
+            default=DiffusCalculationType.isothermal,
```

Comment:
Once again, just for the sake of consistency with MCAS
```suggestion
            default=DiffusivityCalculationType.isothermal,
```